src/common/filter/Expressions.cpp

```diff
@@ -619,6 +621,24 @@ const char* FunctionCallExpression::decode(const char *pos, const char *end) {
     return pos;
 }
 
+std::string UUIDExpression::toString() const {
+    return folly::stringPrintf("uuid(%s)", field_->c_str());
+}
+
+OptVariantType UUIDExpression::eval() const {
+     auto client = context_->storageClient();
+     auto space = context_->space();
+     auto uuidResult = client->getUUID(space, *field_).get();
```

Comment:
Please talk about the sync way with @dutor ,  is it ok in Expression?

Reply:
@dangleptr IMO, such a way to do evaluation asynchronously is more generic in function call.

Reply:
BTW. From discussion above, @darionyaphet will introduce the async way in another PR.